src/lib/Core.jsx

```diff
@@ -368,7 +368,11 @@ function Core({
       formatTime(seconds)
     }${seconds}`;
   };
-
+  const submitQuiz = () => {
+    setIsRunning(false);
+    setEndQuiz(true);
+    getUnansweredQuestions();
+  }
```

Comment:
the code is same as `handleTimeUp`. don't duplicate it. revise the function and use for both case.